wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -150,11 +150,11 @@ public function sendReset($uId, $email)
             'login'
         );
 
-        $subject  = "Set Password";
-        $message  = __('Someone requested that the password be reset for the following account:') . "\r\n\r\n";
-        $message .= __('If this was a mistake, just ignore this email and nothing will happen.') . "\r\n\r\n";
-        $message .= __('To set your password, visit the following address:') . "\r\n\r\n";
-        $message .=  $uniqueUrl;
+        $subject = __("Invitation to collaborate on GC Articles", "cds-snc");
+        $message = __('Someone has invited this email to collaborate on a GC Articles collection site.', "cds-snc") . "\r\n\r\n";
```

Comment:
Do we know who is sending them the email? If so, we could replace the "Someone". 

Although, seems like this works with or without it.

Reply:
We should have the email of the person logged in but I'll skip that in this PR.